<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BachSearchBox</class>
 <widget class="QWidget" name="BachSearchBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="2" column="0">
       <widget class="QGroupBox" name="DateRangeBox">
        <property name="title">
         <string>Date Range</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="margin">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="DR_Taken_From_CB">
           <property name="text">
            <string>Taken From</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDateTimeEdit" name="DR_Taken_From_Value">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="displayFormat">
            <string>yyyy-MM-dd</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="DR_Imported_From_CB">
           <property name="text">
            <string>Imported From</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QDateTimeEdit" name="DR_Imported_From_Value">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="displayFormat">
            <string>yyyy-MM-dd</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="DR_Taken_To_CB">
           <property name="text">
            <string>Taken To</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDateTimeEdit" name="DR_Taken_To_Value">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="displayFormat">
            <string>yyyy-MM-dd</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="DR_Imported_To_CB">
           <property name="text">
            <string>Imported To</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QDateTimeEdit" name="DR_Imported_To_Value">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="displayFormat">
            <string>yyyy-MM-dd</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QGroupBox" name="ExifBox">
        <property name="title">
         <string>Exif Info</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="margin">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QLineEdit" name="EI_Value"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="ImageDetailsBox">
        <property name="title">
         <string>Image Details</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="margin">
          <number>2</number>
         </property>
         <item row="0" column="1">
          <widget class="QComboBox" name="ID_Width_OP">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>&gt;</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>=</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>&lt;</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QComboBox" name="ID_Height_OP">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>&gt;</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>=</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>&lt;</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QSpinBox" name="ID_Height_Value">
           <property name="maximum">
            <number>128000</number>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSpinBox" name="ID_Width_Value">
           <property name="maximum">
            <number>128000</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Width</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QGroupBox" name="CameraInfoBox">
        <property name="title">
         <string>Camera Info</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="margin">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>ISO</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="CI_ISO_OP">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>&gt;</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>=</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>&lt;</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QDoubleSpinBox" name="CI_ISO_Value">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="maximum">
            <double>25600.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>ShutterSpeed (1/'s)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QComboBox" name="CI_Shutter_OP">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>&gt;</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>=</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>&lt;</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QDoubleSpinBox" name="CI_Shutter_Value">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="maximum">
            <double>8000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Lens (mm)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="CI_Lens_OP">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>&gt;</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>=</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>&lt;</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDoubleSpinBox" name="CI_Lens_Value">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <double>6000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Aperture (1/f)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QComboBox" name="CI_Aperture_OP">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>&gt;</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>=</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>&lt;</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QDoubleSpinBox" name="CI_Aperture_Value">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>160.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="FilePathBox">
           <property name="title">
            <string>File Path</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="margin">
             <number>2</number>
            </property>
            <item row="0" column="0">
             <widget class="QLineEdit" name="FP_Value"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="ProjectBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Project</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QComboBox" name="Proj_Value">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <item>
               <property name="text">
                <string>hf1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>hf2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>fury</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="KeywordBox">
           <property name="title">
            <string>Keyword</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="KW_Value"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="KW_Has_Value">
           <property name="text">
            <string>Has Keyword</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>DR_Taken_From_CB</sender>
   <signal>clicked(bool)</signal>
   <receiver>DR_Taken_From_Value</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DR_Taken_To_CB</sender>
   <signal>clicked(bool)</signal>
   <receiver>DR_Taken_To_Value</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DR_Imported_From_CB</sender>
   <signal>clicked(bool)</signal>
   <receiver>DR_Imported_From_Value</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DR_Imported_To_CB</sender>
   <signal>clicked(bool)</signal>
   <receiver>DR_Imported_To_Value</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
